tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -100,13 +100,20 @@ def get_hwm_for_topic(topic):
         # - How mane transactions for the node
 
         # Prepare topics
-        topics_per_node = 4
+        total_workloads = 4
+        unique_topics = False
```

Comment:
hard coded to false? You meant to parameterize the test?

Reply:
Yes, but skipped this until development of multi-node test.